scene/3d/physics/collision_shape_3d.h

```diff
@@ -43,11 +43,11 @@ class CollisionShape3D : public Node3D {
 	uint32_t owner_id = 0;
 	CollisionObject3D *collision_object = nullptr;
 
-#ifdef DEBUG_ENABLED
 	Color debug_color;
 	bool debug_fill = true;
 
 	Color _get_default_debug_color() const;
+#ifdef DEBUG_ENABLED
```

Comment:
```suggestion
	Color _get_default_debug_color() const;

#ifdef DEBUG_ENABLED
```

Reply:
@dalexeev You want me to make the _get_default_debug_color a DEBUG_ENABLED only method?  And I presume all of the callers of that method also DEBUG_ENABLED only?

Reply:
Oh wait.  It says you approved this changes.  Should I make a change or are we all good?

Reply:
I'll just go change it quick

Reply:
I just suggested adding a blank line for better readability.

Reply:
Oh I see.  I thought you were suggesting I move _get_default_debug_color into ``DEBUG_ENABLED``.  I'm actually a fan of doing this as the constructor for CollisionShape2D and CollisionShape3D would no longer need to execute:

```
debug_color = _get_default_debug_color();
```

I expect the runtime overhead is negligeable for the most part.  But if users are spawning "mobs" / millions of colliders, constantly calling ``_get_default_debug_color`` may start to add some unessary overhead on release builds.

So it does make sense to limit this to debug builds.

Reply:
ehhhh or not.  I may be overthinking it.  I'll just go add the extra readability space you suggested

Reply:
@dalexeev There we go!  I fixed the space! :)  Let me know if there is anything else :)